dubbo-serialization/dubbo-serialization-protostuff/src/main/java/org/apache/dubbo/common/serialize/protostuff/ProtostuffObjectOutput.java

```diff
@@ -15,41 +15,48 @@
  * limitations under the License.
  */
 
-package org.apache.dubbo.common.serialize.protobuf;
+package org.apache.dubbo.common.serialize.protostuff;
 
-import io.protostuff.*;
+import io.protostuff.LinkedBuffer;
+import io.protostuff.ProtobufIOUtil;
+import io.protostuff.Schema;
 import io.protostuff.runtime.RuntimeSchema;
 import org.apache.dubbo.common.serialize.ObjectOutput;
-import org.apache.dubbo.common.serialize.protobuf.utils.WrapperUtils;
+import org.apache.dubbo.common.serialize.protostuff.utils.WrapperUtils;
 
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
 
-public class ProtobufObjectOutput implements ObjectOutput {
+public class ProtostuffObjectOutput implements ObjectOutput {
 
+    private LinkedBuffer buffer = LinkedBuffer.allocate();
```

Comment:
This looks like a bug in a multi-threaded environment - change it to thread-local?